.eslintrc.js

```diff
@@ -9,5 +9,58 @@ module.exports = {
 				'node/no-unpublished-import': 0,
 			},
 		},
+		{
+			files: ['**/*.js', '**/*.vue'],
```

Comment:
`files: ['**/*.js', '**/*.vue']` with current run script [eslint --ext .js,.vue src](https://github.com/nextcloud/spreed/blob/373c63e3ccfd03d845a7cf5ee4c7b63bcbf065cb/package.json#L14) means "all files". So after adding `.js` all these rules are not overrides but just rules.

```js
	rules: {
		'import/newline-after-import': 1,
		'import/order': [
			'warn',
			{
				groups: ['builtin', 'external', 'internal', ['parent', 'sibling', 'index'], 'unknown'],
				pathGroups: [
					{
						// group all style imports at the end
						pattern: '{*.css,*.scss}',
						patternOptions: { matchBase: true },
						group: 'unknown',
						position: 'after',
					},
					{
						// group material design icons
						pattern: 'vue-material-design-icons/**',
						group: 'external',
						position: 'after',
					},
					{
						// group @nextcloud imports
						pattern: '@nextcloud/{!(vue),!(vue)/**}',
						group: 'external',
						position: 'after',
					},
					{
						// group @nextcloud/vue imports
						pattern: '{@nextcloud/vue,@nextcloud/vue/**}',
						group: 'external',
						position: 'after',
					},
					{
						// group project components
						pattern: '*.vue',
						patternOptions: { matchBase: true },
						group: 'external',
						position: 'after',
					},
				],
				pathGroupsExcludedImportTypes: ['@nextcloud', 'vue-material-design-icons'],
				'newlines-between': 'always',
				alphabetize: {
					order: 'asc',
					caseInsensitive: true,
				},
				warnOnUnassignedImports: true,
			},
		],
	},

	overrides: [
		{
			files: ['**/*.spec.js'],
			rules: {
				'node/no-unpublished-import': 0,
			},
		},
	],
```